tooling/eslint/nextjs.js

```diff
@@ -0,0 +1,18 @@
+import nextPlugin from "@next/eslint-plugin-next";
+
+/** @type {Awaited<import('typescript-eslint').Config>} */
+export default [
+  {
+    files: ["**/*.ts", "**/*.tsx"],
+    plugins: {
+      "@next/next": nextPlugin,
+    },
+    rules: {
+      ...nextPlugin.configs.recommended.rules,
+      ...nextPlugin.configs["core-web-vitals"].rules,
+      // TypeError: context.getAncestors is not a function
```

Comment:
sorry, could i get more context on what this is?

Reply:
eslint v9 some slight incompatibilities with the current plugins. So that rule is ignored. But actually i fixed in a later PR haha